arangod/Cache/FrequencyBuffer.h

```diff
@@ -32,7 +32,8 @@
 #include <utility>
 #include <vector>
 
-#include "Basics/SharedPRNG.h"
+#include "ApplicationFeatures/SharedPRNGFeature.h"
+#include "Basics/debugging.h"
```

Comment:
```suggestion
```
I did not find anything that would require this include. Did I miss something?

Reply:
Yes, TRI_ASSERT(...) was used in this file (even before my change), so following the include-what-you-use scheme I added this include here.